styles/utility.css

```diff
@@ -700,4 +839,116 @@ textarea {
   hr {
     margin-top: 70px;
   }
+
+  .modal {
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    overflow: auto;
+    background-color: rgba(0, 0, 0, 0.1);
+    z-index: 2;
+    animation: modalopen 1s ease-out;
+    display: none;
+  }
+
+  .modal-container {
+    display: flex;
+    position: absolute;
+    justify-content: space-evenly;
+    margin: 60px 16px 0 16px;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    width: 100%;
+    width: 85%;
+    height: 100%;
+    padding: 18px 16px;
+  }
+
+  .modal-wrapper {
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 18px;
+    width: 75%;
+    height: 85vh;
+    padding: 5px 5px 10px 5px;
+    margin: 0 auto;
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+  }
+
+  .project-image img {
+    margin-top: 20px;
+    content: url('../images/SnapshootPortfolioDesktop.png');
+    width: 100%;
+  }
+
+  .close i {
+    position: absolute;
+    color: #ebecf0;
+    float: right;
+    font-size: 28px;
+    font-weight: bold;
+    padding-top: 5px;
+    padding-bottom: 10px;
+    padding-left: 325px;
+  }
```

Comment:
- [ ] Please as requested by the previous code reviewer, the `close button` is supposed to be inside the popup, please kindly fix that.

Reply:
Hi @t-yanick 
The close button is now inside the popup